insights/specs/datasources/ps.py

```diff
@@ -76,3 +78,62 @@ def ps_eo_cmd(broker):
         return DatasourceProvider('\n'.join(data), relative_path='insights_commands/ps_eo_cmd')
 
     raise SkipComponent()
+
+
+@datasource(LocalSpecs.ps_eo_args, HostContext)
+def jboss_runtime_versions(broker):
```

Comment:
I don't see a parser for this datasource, is this just an analytics spec/datasource?

Reply:
This is for subscription watch service, I am not sure if need parser. Another thing, I want to collect data, then decide for parser writing.

Reply:
@chenlizhong - please open an issue for this spec to the core-asserts repo to get it approved.